src/locale/locales/fr/messages.po

```diff
@@ -5423,19 +5047,19 @@ msgstr "Afficher l’entrée de débogage"
 
 #: src/components/ReportDialog/SelectReportOptionView.tsx:132
 msgid "View details"
-msgstr ""
+msgstr "Voir les détails"
 
 #: src/components/ReportDialog/SelectReportOptionView.tsx:127
 msgid "View details for reporting a copyright violation"
-msgstr ""
+msgstr "Voir les détails pour signaler une violation du droit d’auteur"
 
 #: src/view/com/posts/FeedSlice.tsx:99
 msgid "View full thread"
 msgstr "Voir le fil de discussion entier"
 
 #: src/components/moderation/LabelsOnMe.tsx:51
 msgid "View information about these labels"
-msgstr ""
+msgstr "Voir les informations sur ces labels"
```

Comment:
Fun fact: `label` is a thing in French, but doesn't have the same usage. Well, hm, that's debatable, but I don't think it is used in a negative way: a `label` is typically a good thing you are looking for, applying for, etc. `Étiquettes` is more neutral in that matter.

```suggestion
msgstr "Voir les informations sur ces étiquettes"
```